### DISCUSSION TITLE
install.exe gets caught by Windows's SmartScreen

### DISCUSSION THREAD

**Discussion by lilyremigia** [ppy/osu#15067] [Q&A] [answered]
![Microsoft Defender SmartScreen "protecting" me](https://user-images.githubusercontent.com/13798839/137020472-03d8552b-eaa4-46d9-9daa-7871313234f1.png)

**Comment by bdach** [comment]
mind translating? i am not capable of reading hungarian.

**Reply by lilyremigia** [reply]
![English version](https://user-images.githubusercontent.com/13798839/137031989-4d75860c-ab66-4cbe-b5da-2a631a4a31bf.png)
Basically Windows not trusting this since (I'm assuming) it has been only signed by an OV and not an EV, and hasn't been downloaded and run enough times.

**Comment by peppy** [comment, marked as answer]
I've submitted a false positive report to Microsoft.

In general though, there's not much we can do about this but wait for microsoft to begin to like us again.
